### PR TITLE
docs: better FW config

### PR DESCRIPTION
**Pull request by QuentinN42**
https://docs-git-feat-repeater-ips-escape-tech.vercel.app/platform/enterprise/repeater#configure-your-firewall

### CODE REVIEW

File: docs/03-platform/03-enterprise/07-repeater.mdx [hunk 1]
```diff
@@ -104,23 +104,20 @@ users:
...
-| `52.6.17.196`       | `TCP`/`HTTP`  | `80`  |
-| `52.6.17.196`       | `TCP`/`HTTP`  | `443`  |
-| `52.5.35.215`       | `TCP`/`HTTPS` | `80` |
-| `52.5.35.215`       | `TCP`/`HTTPS` | `443` |
-| `34.225.45.162`       | `TCP`/`HTTPS` | `80` |
-| `34.225.45.162`       | `TCP`/`HTTPS` | `443` |
+| Address          | Protocol      | Port  |
+| ---------------- | ------------- | ----- |
+| `51.159.205.221` | `TCP`/`HTTP`  | `80`  |
+| `51.159.205.221` | `TCP`/`HTTPS` | `443` |
```
Comment (Gby56) [inline docs/03-platform/03-enterprise/07-repeater.mdx:120]: wrong ips no ? these resolve to scaleway still

**Review by Gby56 — CHANGES_REQUESTED** [review]

File: docs/03-platform/03-enterprise/07-repeater.mdx (hunk 1 above)
Comment (QuentinN42) [reply docs/03-platform/03-enterprise/07-repeater.mdx:120]: Yes this is the tools cluster

Comment (Gby56) [reply docs/03-platform/03-enterprise/07-repeater.mdx:120]: Ok so better to keep the ips I recently updated in the docs then, these are AWS

Comment (QuentinN42) [reply docs/03-platform/03-enterprise/07-repeater.mdx:128]: Added the current IPs in the doc.

**Review by Gby56 — APPROVED** [review]

**Pull request merged**